### DISCUSSION TITLE
Open on MacOS

### DISCUSSION THREAD

**Discussion by stephenebrandt** [GrandOrgue/GoOdf#104] [Q&A] [answered]
How do I open this on MacOS?

**Comment by stephenebrandt** [comment]
I searched in the wrong area and found my answer

**Reply by larspalo** [reply, marked as answer]
@stephenebrandt As of version 0.13.0 there are now released builds for MacOS with both Intel and arm64 architectures available.

**Reply by stephenebrandt** [reply]
Thank you!

**Reply by stephenebrandt** [reply]
@larspalo I just realized it doesn't work on my MacOS. I'm running the latest, Sonoma 14.4.1.

**Reply by larspalo** [reply]
@stephenebrandt And you're sure that you're using the correct version of the build for your computer architecture (x86_64 vs arm64)? Do you get any warning or error messages at all?

**Comment by MStraeten** [comment]
the app seems to be just ad-hoc signed (i.e. without a developer certificate and notarization), so you need to help sonoma to accept it as ok:
1. run the app via context menu and confirm the message
2. run from terminal `xattr -c  /Applications/GoOdf.app` (or whereever you copied the app)

**Reply by stephenebrandt** [reply]
I had accepted it. For some reason it wasn't showing the main GUI screen whenever I tried creating a new organ or opening one. I did something and now it's showing. So thank you again.
